src/constant.ts

```diff
@@ -30,4 +30,5 @@ export const WHITELISTED_ORIGINS = [
   'https://google.com',
   'https://front-web-platform-dev',
   'https://front-b2b-api-test.azurewebsites.net',
+  'https://link.trustwallet.com'
```

Comment:
Have you tested this change on both platforms? I'm pretty sure it will break deeplinking to trust wallet, since the trust webpage will open in the SDK, and user will have no way of navigating back.